config/clusteroperator/core_clusteroperator.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: core
+spec: {}
+status:
+  versions:
+  - name: operator
+    version: "0.0.1-snapshot"
```

Comment:
What's the story here? Does `0.0.1-snapshot` have any significance? Is there a reason we statically set this as opposed to leaving it empty here and setting in the controller?

Reply:
Looking at https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/dev/clusteroperator.md#what-should-be-the-contents-of-clusteroperator-custom-resource-in-manifests, it sounds like there's logic that looks for this status version and does a copy/replace operation with what's present in the release payload. Not too sure whether this is directly needed in the static YAML manifest, but I've seen a bunch of payload components specify this in their CVO manifests directory.